participants.md

```diff
@@ -1,2 +1 @@
-Sajal
-Vivek
+tej
```

Comment:
Deleted my work ?

Reply:
try to get merged without raising any conflict